YellowNotes.Api/YellowNotes.Api.csproj

```diff
@@ -4,9 +4,21 @@
     <ProjectReference Include="..\YellowNotes.Core\YellowNotes.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Controllers\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.1" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.6.0" />
+  </ItemGroup>
+
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>YellowNotes.Api</RootNamespace>
+    <AssemblyName />
+    <StartupObject>YellowNotes.Api.Program</StartupObject>
+    <AutoGenerateBindingRedirects></AutoGenerateBindingRedirects>
```

Comment:
do we need this? this is for what? I'm asking because I've never seen it before.

Reply:
It has generated automatically by accident. I've removed it